doc/flatpak-metadata.xml

```diff
@@ -253,6 +268,54 @@
                                 Available since 0.3.
                             </para></listitem></varlistentry>
 
+                            <varlistentry><term><option>host-os</option></term>
+                            <listitem><para>
+                                The host operating system's libraries,
+                                executables and static data from
+                                <filename>/usr</filename>
```

Comment:
Note that this includes `/usr/local`, unless `/usr/local` is a symlink to somewhere else (as it is likely to be on OSTree systems).

Do we want this? If not, we could consider having this pseudocode:

```
if /usr/local is a directory and is not a symbolic link and not expose_host_usrlocal:
    mount a tmpfs over it
```

Reply:
I think having /usr/local is fine. Its still the "os" rather than personal data.